server/src/main/java/io/spine/server/event/MessageEnrichment.java

```diff
@@ -146,7 +146,8 @@ private void setFields(Message.Builder builder, S sourceMessage, C context) {
                                                              Field.getFieldClass(targetField)));
                 EnrichmentFunction fieldEnrichment = function.get();
                 @SuppressWarnings("unchecked"
```

Comment:
Please see how a long comment to such a suppression is formatted in `RecordBasedRepository`.